packit_service/worker/handlers/forges.py

```diff
@@ -90,3 +96,119 @@ def run(self) -> TaskResults:
+            event=event,
+        )
+        self.sender_login = self.data.actor
+        self.comment = self.data.event_dict.get("comment")
+        self._issue = None
+
+    @property
+    def issue(self):
+        if not self._issue:
+            self._issue = self.project.get_issue(self.data.issue_id)
+        return self._issue
+
+    def pre_check(self) -> bool:
+        """
+        Checks whether the Packit verification command is placed in
+        packit/notifications repository in the issue our service created.
+        """
+        if not (
+            self.project.namespace == "packit" and self.project.repo == "notifications"
+        ):
+            logger.debug(
+                "Packit verification comment command not placed in packit/notifications repository."
+            )
+            return False
+
+        issue_author = self.issue.author
+        if (
+            self.service_config.deployment == Deployment.prod
+            and issue_author != "packit-as-a-service[bot]"
+        ) or (
+            self.service_config.deployment == Deployment.stg
+            and issue_author != "packit-as-a-service-stg[bot]"
+        ):
+            logger.debug(
+                f"Packit verification comment command placed on issue with author "
+                f"other than our app: {issue_author}"
+            )
+            return False
+
+        return True
+
+    def run(self) -> TaskResults:
+        """
+        Discover information about organization/user which wants to verify the FAS account.
+        Allowlist automatically if mapping from github username to FAS account can prove a match.
+
+        Returns:
+            TaskResults
+        """
+        logger.debug(
+            f"Going to run verification of FAS account triggered by comment:"
+            f" {self.comment}"
+        )
+        # e.g. User Bebaabeni needs to be approved.
+        account_type, account_login, _ = self.issue.title.split(maxsplit=2)
+        namespace = f"github.com/{account_login}"
+        command_parts = get_packit_commands_from_comment(
+            self.comment, self.service_config.comment_command_prefix
+        )
+        # we expect ["verify-fas", "fas-account"]
+        if len(command_parts) != 2:
+            msg = "Incorrect format of the Packit verification comment command."
+            logger.debug(msg)
+            self.issue.comment(
+                f"{msg} The expected format: `/packit verify-fas my-fas-account`"
+            )
+            return TaskResults(success=False, details={"msg": msg})
+
+        fas_account = command_parts[1]
+
+        allowlist = Allowlist(service_config=self.service_config)
+        if allowlist.is_approved(namespace):
+            msg = (
+                f"Invalid namespace! {account_type} `{account_login}` already approved."
+            )
+            logger.debug(msg)
+            self.issue.comment(msg)
+            return TaskResults(success=False, details={"msg": msg})
```

Comment:
I am trying to understand how could this happen (someone did it manually on CLI?) and if it's actually a problem. Sounds like someone resolved the ask in the background :D and is not an error.

Reply:
yes, that's true, it is not very likely. So do you propose to not handle this situation at all or just referencing to change `success` to True?

Reply:
A counterquestion for you: if you saw a sentry issue for this, would you investigate it?

I don't personally think we need a sentry issue here, the comment in the issue should be sufficient. Can we also make it more polite, please? Something like `Namespace {account_login} was already approved`.

Reply:
> A counterquestion for you: if you saw a sentry issue for this, would you investigate it?

Ok, totally forgot that these come to Sentry :D makes sense, will change it including the message, thanks!